shell/pages/account/index.vue

```diff
@@ -165,15 +173,14 @@ export default {
           type="button"
           class="btn role-primary"
           data-testid="account_change_password"
-          @click="$refs.promptChangePassword.show(true)"
+          @click="showChangePasswordDialog"
         >
           {{ t("accountAndKeys.account.change") }}
         </button>
       </div>
     </div>
-    <PromptChangePassword ref="promptChangePassword" />
 
-    <hr role="none">
+    <hr>
```

Comment:
Why did we remove the `role="none"` in this PR? Didn't we recently add this throughout dashboard in #14121?